docs/UpdatingProduction.md

```diff
@@ -82,7 +82,9 @@ Verify IAM Policy, Groups and Roles are the latest.  Master IAM scripts are loca
 Make sure your AWS_CREDENTIALS is set for the dev account
 ```shell
 $ cd boss-manage.git/bin
-$ ./iam_utils.py export
+$ ./iam_utils.py production.boss export groups
```

Comment:
can also be written `./iam_utils.py production.boss export groups roles policies`